src/sgame/sg_main.cpp

```diff
@@ -1974,7 +1974,7 @@ void CheckExitRules()
 	{
 		//humans win
 		level.lastWin = TEAM_HUMANS;
-		trap_SendServerCommand( -1, "print_tr \"" N_("Humans win") "\"" );
+		trap_SendServerCommand( -1, "print_tr " QQ( N_("Humans win") ) );
```

Comment:
Just pointing at this example, but there's a bit more of that:

This could probably use `const char *BG_TeamName( int team );` or `const char *BG_TeamNamePlural( int team );` that is:

```
trap_SendServerCommand( -1, "print_tr " QQ( N_( va( "%s win", BG_TeamNamePlural( level.lastWin ) ) ) ) );
```

Which could lead to the removal of more lines of code and strings to translate. One have to check if BG_TeamNamePlural and BG_TeamName returns are translated or not, but I doubt they are.

Reply:
Hu no you can't assume every language has subject before verb, neither every language doesn't have any declension for a word being subject or not.

I'm splitting between sentences, not within sentences, because every language can translate a sentence.